controller/src/main/java/org/jboss/as/controller/OperationContext.java

```diff
@@ -514,7 +527,9 @@ default ServiceTarget getServiceTarget() throws UnsupportedOperationException {
     /**
      * Create a new resource, relative to the executed operation address.  Since only one operation
      * may write at a time, this operation may block until other writing operations have completed.
-     *
+     * <p>
+     * This method performs an authorization check to ensure that the caller has the necessary permissions
+     * to create the resource.
```

Comment:
Nitpick: we usually leave an empty space in Javadoc between text and @ params.

Reply:
Fixed

Reply:
There are 4-5 more occurrences of that in this change still...